frontend/src/components/VFilters/VSearchGridFilter.vue

```diff
@@ -74,9 +73,6 @@ export default defineComponent({
       default: true,
     },
   },
-  emits: {
-    close: defineEvent(),
-  },
```

Comment:
Why was this removed?

Reply:
`emits` is supposed to document and set types for events emitted by a component but this event wasn't actually emitted anywhere.